index.html

```diff
@@ -275,10 +275,17 @@ <h3>
                 <li>Test Suites</li>
               </ul>
             </li>
+            <li>Other VC Data Integrity proof types:
+              <ul>
+                <li>VC Data Integrity - <a href="https://or13.github.io/lds-pgp2021"></a>PGP</li>
+                <li>VC Data Integrity - <a href="https://ns.did.ai/suites/multikey-2021"></a>Multbase / MultiCodec</li>
```

Comment:
```suggestion
```

Reply:
Yes, confirmed... and to be crystal clear so comment thread is not used against the VCWG: The reason I'm recommending the removal of this line item is because multikey, which uses multibase and multicodec, is already a part of a normative deliverable the group has identified in the current charter proposal. To say that it is both a normative and non-normative deliverable would be confusing. We should state clearly that it is a normative deliverable (as a part of the verification method definition in the Data Integrity specification).

I'll respond to your commentary above when I get some free cycles, but the TL;DR is that I believe we're aligned on the end goal here (or some variation of it).